src/gt4py/next/ffront/dialect_parser.py

```diff
@@ -106,11 +106,11 @@ def get_location(self, node: ast.AST) -> SourceLocation:
 
         # `FixMissingLocations` ensures that all nodes have the location attributes
         assert hasattr(node, "lineno")
-        line = node.lineno + line_offset if node.lineno is not None else None
+        line = node.lineno + line_offset
```

Comment:
let's see if that works (mypy complain because line, colum are not Optional)

Reply:
Ah great they fixed this in python.